www/include/reporting/dashboard/csvExport/csv_HostLogs.php

```diff
@@ -62,111 +60,175 @@
 
 $centreon = $_SESSION["centreon"];
 
-/*
- * Checking session
- */
+// Checking session
 $sid = session_id();
 if (!empty($sid) && isset($_SESSION['centreon'])) {
     $oreon = $_SESSION['centreon'];
-    $query = "SELECT user_id FROM session WHERE user_id = '".$pearDB->escape($oreon->user->user_id)."'";
-    $res = $pearDB->query($query);
+    $res = $pearDB->query(
+        "SELECT user_id FROM session " .
+        "WHERE user_id = '" . $pearDB->escape($oreon->user->user_id) . "'"
```

Comment:
In this case, using a prepare statement is hightly recommended

Reply:
Refacto the whole code is out of my scope...  ;)